### DISCUSSION TITLE
Your Team encountered an unknown problem. Please reach out to our support team for details. Learn More

### DISCUSSION THREAD

**Discussion by reenamyka** [vercel/community#6840] [Help] [answered]
### Summary

I got this error when I deployed my next.js project to vercel

### Steps to Reproduce

Login via github -> import repository -> deploy desired repository

**Comment by reenamyka** [comment]
Help @amyegan

**Reply by reenamyka** [reply]
@swarnava

**Reply by Bhaney44** [reply]
See: https://github.com/orgs/vercel/discussions/6027

We are still waiting on a reply.

**Comment by amyegan** [comment, marked as answer]
https://github.com/vercel/community/discussions/6027#discussioncomment-9439020
